components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -83,25 +85,55 @@ export default {
 
 export const Default = ActionButtonGroup.bind({});
 Default.args = {
-	label: "More",
+	label: "Edit",
+	iconName: "Edit",
 };
 
 export const Emphasized = ActionButtons.bind({});
```

Comment:
The docs site has an "emphasized quiet" variant. I don't think I am really missing it- it's kind of weird to have a quiet variant, but then it be "emphasized." Maybe just a blurb in the quiet story docs? Something like "A quiet action button that is selected can also have emphasized styling." I don't know, just a thought. If we can figure out like, a class name that gets added or removed, we can always note that in the description too!